docs/user_documentation.md

```diff
@@ -225,15 +225,16 @@ Setting up Relativity Trace
+Trace has a six-step process. All new documents go through first three steps (`1 – New`, `2 – Indexed`, `3 – Term Searched`). The next three steps (`4 – Normalized`, `5 – Ready for Rule Analysis`, `6 – Alert Rules Complete`) depend on configuration in `Rule Evaluation` task.  Status is tracked in the `Trace Document Status` field.
 
-Trace has a three-step process that all new documents go through. This status is tracked in the `Trace Document Status` field.
-
-|             | **Step 1**                                                   | **Step 2**                                    | **Step 3**                                                   |
-| ----------- | ------------------------------------------------------------ | --------------------------------------------- | ------------------------------------------------------------ |
-| **Status:** | **NEW**                                                      | **INDEXED**                                   | **TERM SEARCHED**                                            |
-| Overview:   | Documents that are brand new and are part of `Trace All  Documents` saved search | Documents that have been successfully indexed | Documents that have been successfully searched by Term Searching task |
-
-> **NOTE:** Rule evaluation (including tagging documents to the Rule) is executed by the Rule Evaluation task and is outside of the core Trace Document Flow.
+| Step | Status:                 | Overview:                                                    |
+| :--: | ----------------------- | ------------------------------------------------------------ |
+|  1   | NEW                     | Documents that are brand new and are part of `Trace All Documents` saved search. |
+|  2   | INDEXED                 | Documents that have been successfully indexed.               |
+|  3   | TERM SEARCHED           | Documents that have been successfully searched by Term Searching task. |
+|  4   | NORMALIZED              | Documents which are returned by `Normalized Saved Search` configured in `Rule Evaluation` task along with their family (documents are marked as `Normalized` only if whole family is in status `Term Searched`). If `Normalized Saved Search` is not configured, all trace documents, which have family term searched, are getting normalized. |
+|  5   | READY FOR RULE ANALYSIS | Documents which were in `Normalized` status before along with their family and whose parent documents do not belong to `Omit from Alert Rules Saved Search` configured in `Rule Evaluation` task. |
+|  6   | ALERT RULES COMPLETE    | Documents which are returned by `Omit from Alert Rules Saved Search` configured in `Rule Evaluation` task along with their family (documents are marked as `Alert Rules Complete` only if they were in `Normalized` status before). If ``Omit from Alert Rules Saved Search`` is not configured, none of the documents is marked with `Alert Rules Complete` status. Those documents are not tagged by Alert Rules. |
 
```

Comment:
Maybe "Normalized Saved Search" and "Omit from Alert Rules Saved Search" should be described somewhere else - how they should be setup and when they should be setup.